<ui version="4.0" >
 <class>StartPracticeWidget</class>
 <widget class="QWidget" name="StartPracticeWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="testTypeGroupBox" >
     <property name="title" >
      <string>Type of Test</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="WrittenRadio" >
        <property name="text" >
         <string>Written</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="SynonymRadio" >
        <property name="text" >
         <string>Synonym</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="MultipleChoiceRadio" >
        <property name="text" >
         <string>Multiple Choice</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QRadioButton" name="AntonymRadio" >
        <property name="text" >
         <string>Antonym</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QRadioButton" name="GrammarRadio" >
        <property name="text" >
         <string>Grammar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QRadioButton" name="ParaphraseRadio" >
        <property name="text" >
         <string>Paraphrase</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QRadioButton" name="ExampleRadio" >
        <property name="text" >
         <string>Example Sentences</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="GrammarGroupBox" >
     <property name="title" >
      <string>Grammar</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="ConjugationCheckBox" >
        <property name="text" >
         <string>Conjugation of Verbs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ComparisonAdjectiveCheckBox" >
        <property name="text" >
         <string>Comparison of Adjectives</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ComparisonAdverbCheckBox" >
        <property name="text" >
         <string>Comparison of Adverbs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ArticleCheckBox" >
        <property name="text" >
         <string>Articles of Nouns</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="languageGroupBox" >
     <property name="title" >
      <string>Languages:</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>From:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>To:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QListWidget" name="LanguageFromList" />
        </item>
        <item row="1" column="1" >
         <widget class="QListWidget" name="LanguageToList" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
